### PR TITLE
Add email notification for ONT run events

### PR DESCRIPTION
**Pull request by kjsanger**
Add dependency on Partisan (for baton JSON IO - no requirement for iRODS access or baton itself).

Add version() function to report the notification pipeline version for ONT.

Add necessary ONT tables to MLWH ORM.

Add dev Dockerfile and compose file.

Rename some QC event-specific resources to have "QC" in the name, to allow ONT event-specific resources to be added.

### CODE REVIEW

File: tests/ont/conftest.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import requests
+from npg.conf import IniData
+from requests import HTTPError
+
+from npg_notify.ont.event import PORCH_CONFIG_FILE_SECTION
+from npg_notify.ont.porch import Pipeline
+
+ont_test_config = Path("./tests/data/ont_event_app_config.ini")
+
+
```
Comment (kjsanger) [inline tests/ont/conftest.py:31]: Missing test annotation

File: tests/ont/test_generate_email.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+from pytest import mark as m
+
+from npg_notify.ont.event import ContactEmail, EventType
+
+
+@m.describe("Generate ONT email")
+class TestGenerateONTEmail:
+
+    def test_serialize_deserialize_event(self):
```
Comment (kjsanger) [inline tests/ont/test_generate_email.py:11]: Missing test annotation

File: tests/ont/test_generate_email.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            json.loads(json.dumps(event1.to_serializable()))
+        )
+        assert event2.experiment_name == expt
+        assert event2.instrument_slot == slot
+        assert event2.flowcell_id == flowcell_id
+        assert event2.path == path
+        assert event2.event == event_type
+
+    @m.context("When an ONT email is generated")
+    @m.it("Has the correct subject")
```
Comment (kjsanger) [inline tests/ont/test_generate_email.py:36]: Test annotation needs expanding (the test checks more than just the subject)

File: src/npg_notify/ont/event.py [hunk 4]
```diff
@@ -0,0 +1,463 @@
...
+        session: An open MLWH DB session.
+        domain: A network domain name to use when sending email. The main will be sent
+           from mail.<domain> with <user>@<domain> in the From: header.
+
+    Returns:
+        The number of tasks processed, the number of tasks that succeeded, and the
+        number of errors encountered.
+    """
+    np = ns = ne = 0
+    batch_size = 1
```
Comment (kjsanger) [inline src/npg_notify/ont/event.py:283]: The batch size should be increased to the maximum number of emails that it's reasonable to send at one time

**Review by marcomoscasgr — APPROVED** [review]
Run and tested in a new vm. The image for porch-server in docker-compose file should be replaced.

**Pull request merged**